tests/query_interface_test.js

```diff
@@ -0,0 +1,582 @@
+require('./helper');
+
+var expect = require('chai').expect;
+var Sequelize = require('..');
+
+const DataTypes = Sequelize.DataTypes;
+const _ = require('lodash');
+
+describe('QueryInterface', () => {
+  beforeEach(function() {
+    this.sequelize.options.quoteIdenifiers = true;
+    this.queryInterface = this.sequelize.getQueryInterface();
+  });
+
+  afterEach(async function() {
+    await this.queryInterface.dropAllSchemas();
+  });
+
+  describe('dropAllSchema', () => {
+    it('should drop all schema', async function() {
+      await this.queryInterface.dropAllSchemas({
+        skip: [this.sequelize.config.database]
+      });
+      const schemaNames = await this.queryInterface.showAllSchemas();
+      await this.queryInterface.createSchema('newSchema');
+      const newSchemaNames = await this.queryInterface.showAllSchemas();
+      expect(newSchemaNames).to.have.length(schemaNames.length + 1);
+      await this.queryInterface.dropSchema('newSchema');
+    });
+  });
+
+  // Reason: this test doesn't work well locally
```

Comment:
Could you leave a comment explaining why the test doesn't work locally?

Reply:
sure